src/components/memo/Card.vue

```diff
@@ -0,0 +1,73 @@
+<template>
+  <div class="card" :class="{faceUp: card.isFaceUp, found: card.isFound}" @click="flip">
+    <p v-if="card.isFaceUp">{{ value }}</p>
+  </div>
+</template>
+
+<script>
+export default {
+  name: "card",
+  props: {
+    flippedProp: {
```

Comment:
Vi må finne ut hvordan dette er tenkt...

Jeg synes det ser så rart ut med behovet for både flippedProp & flipped, kan de ikke være samme?

Reply:
Bruker du flippedProp?